test/topology/test_topology.py

```diff
@@ -62,3 +67,74 @@ async def test_decommission_node_add_column(manager, random_tables):
     await manager.decommission_node(servers[1])             # Decommission [1]
     await table.add_column()
     await random_tables.verify_schema()
+
+
+@pytest.mark.asyncio
+@pytest.mark.skip(reason="Wait for @slow attribute, #11713")
+async def test_remove_node_with_concurrent_ddl(manager, random_tables):
+    stopped = False
+    ddl_failed = False
+
+    async def do_ddl():
+        nonlocal ddl_failed
+        iteration = 0
+        while not stopped:
+            logger.debug(f'ddl, iteration {iteration} started')
+            try:
+                # If the node was removed, the driver may retry "create table" on another node,
+                # but the request might have already been completed.
+                # The same applies to drop_table.
+
+                await random_tables.add_tables(5, 5, if_not_exists=True)
+                await random_tables.verify_schema()
+                while len(random_tables.tables) > 0:
+                    await random_tables.drop_table(random_tables.tables[-1], if_exists=True)
+                logger.debug(f'ddl, iteration {iteration} finished')
+            except:
+                logger.exception(f'ddl, iteration {iteration} failed')
+                ddl_failed = True
+                raise
+            iteration += 1
+
+    async def do_remove_node():
+        for i in range(10):
+            logger.debug(f'do_remove_node [{i}], iteration started')
+            if ddl_failed:
+                logger.debug(f'do_remove_node [{i}], ddl failed, exiting')
+                break
+            server_ids = await manager.running_servers()
+            host_ids = []
+            for server_id in server_ids:
+                # check that node is alive and get its host_id
+                host_id = await manager.get_host_id(server_id)
+                host_ids.append(host_id)
+            initiator_index = random.randint(0, len(server_ids) - 1)
+            initiator_ip = server_ids[initiator_index]
+            while True:
+                target_index = random.randint(0, len(server_ids) - 1)
```

Comment:
WDYT about
```
initiator_index, target_index = random.sample(range(len(server_ids)), 2)
```
no manual loop, won't enter infinite loop if we accidentally ended up with one server :) (would throw instead)

Reply:
good point, thanks